app/views/content/life-as-a-teacher/explore-subjects/modern-languages/_article.html.erb

```diff
@@ -0,0 +1,149 @@
+      <li>writing creatively to express their own ideas and opinions</li>
+    </ul>
+
+    <p>Themes you'll cover when you teach 14 to 16 year olds (key stage 4) include:</p>
+    <ul>
+      <li>understanding and responding to spoken language</li>
+      <li>communicating and interacting in speech</li>
+      <li>understanding and responding to written language</li>
+      <li>communicating in writing</li>
+    </ul>
+    <p>Explore what teaching modern language lessons would be like at key stage 4:</p>
+      <ul>
+        <li><a href="https://www.thenational.academy/teachers/programmes/french-secondary-ks4-l/units">French lessons</a></li>
+        <li><a href="https://www.thenational.academy/teachers/programmes/german-secondary-ks4-l/units">German lessons</a></li>
+        <li><a href="https://www.thenational.academy/teachers/programmes/spanish-secondary-ks4-l/units">Spanish lessons</a></li>
+      </ul>
+    <p>At A level and AS level (key stage 5), you’ll help pupils develop a deeper knowledge and understanding of:</p>
+      <ul>
+        <li>the society and culture, past and present, of the country or countries where the language is spoken</li>
+        <li>using language spontaneously to initiate communication</li>
+        <li>using pronunciation, morphology and syntax, vocabulary and idiom to communicate accurately and coherently</li>
+        <li>listening and responding to spoken passages</li>
+        <li>reading and responding to a variety of texts</li>
+        <li>understanding the main points, gist and detail from spoken and written material </li>
+        <li>inferring meaning from complex spoken and written material, including factual and abstract content</li>
+        <li>summarising information from spoken and written sources</li>
+      </ul>
+    <p>Teaching pupils at different ages and abilities means you'll always be expanding your knowledge and approach to meet pupils' needs. Once you have qualified teacher status (QTS) you can also teach any other subject, as long as you can show you have enough expertise.</p>
+    </section>
+  <section class="col col-space-m col-720">
+  <h3>Teaching ancient languages</h3>
+  <p>When teaching ancient languages, you'll focus on giving pupils a linguistic foundation for reading comprehension and an appreciation of classical civilisation. Having a foundation in ancient languages may make it easier for pupils to learn modern languages, such as French and Spanish.</p>
+  <p>Learning an ancient language such as ancient Greek can give pupils access to historical literature and philosophy, whereas learning Latin could help improve their skills in biology for example. Knowing ancient languages could open the doors for pupils to work in careers such as archaeology, theology and philosophy. </p> 
+  <p>Find out more about <a href="https://www.gov.uk/government/publications/gcse-ancient-foreign-languages">what pupils will learn when studying ancient languages at GCSE</a>.</p>
```

Comment:
```suggestion
  <p>Find out <a href="https://www.gov.uk/government/publications/gcse-ancient-foreign-languages">what pupils will learn when studying ancient languages at GCSE</a>.</p>
```

The main reason I'm suggesting this is that the full-stop wraps oddly on desktop!

![image](https://github.com/user-attachments/assets/206c8310-1742-4df6-98c3-98796958cc8b)